test_runner/regress/test_timeline_delete.py

```diff
@@ -258,12 +258,12 @@ def test_delete_timeline_exercise_crash_safety_failpoints(
     if check is Check.RETRY_WITH_RESTART:
         env.pageserver.stop()
         env.pageserver.start()
-        if failpoint == "timeline-delete-before-index-deleted-at":
-            # We crashed before persisting this to remote storage, need to retry delete request
 
-            # Wait till tenant is loaded. Shouldnt take longer than 2 seconds (we shouldnt block tenant loading)
-            wait_until_tenant_active(ps_http, env.initial_tenant, iterations=2)
+        # Wait till tenant is loaded. Shouldnt take longer than 2 seconds
+        wait_until_tenant_active(ps_http, env.initial_tenant, iterations=2)
```

Comment:
If I am not misunderstanding the logs, it took more than `2 < x < 3` seconds in the test for the tenant to finish loading, unsure why. Should we allow for a longer timeout here, or just go with the default. It was with real_s3 after all..

Reply:
Done